### PR TITLE
[ADEN-10363] 50:50 experiment clean up

### PR DESCRIPTION
**Pull request by Loczi94**

### CODE REVIEW

File: examples/slots/rotating-slots/index.html [hunk 1]
```diff
@@ -1,517 +0,0 @@
...
-				window.adsQueue.push({
-					id: 'top_leaderboard',
-				});
-			</script>
-		</header>
-		<div class="wrapper">
-			<article class="main">
-				<p></p>
-				<strong>Disable FMR rotation (just the first slot):</strong>
-				<pre>?recirculation_disabled=1</pre>
```
Comment (dmnsobczak) [inline examples/slots/rotating-slots/index.html:45]: Please don't delete this example page. It's useful for normal FMR. We can just remove this parameter :D

File: examples/slots/rotating-slots/script.ts [hunk 2]
```diff
@@ -1,20 +0,0 @@
...
-
-context.extend(customContext);
-context.set('slots.bottom_leaderboard.disabled', false);
-context.set('slots.repeatable_boxad_1.avoidConflictWith', undefined);
-context.set('slots.repeatable_boxad_1.insertBeforeSelector', '#repeatable_boxad_1');
-context.set('slots.repeatable_boxad_1.repeat.limit', 20);
-context.set('slots.repeatable_boxad_1.repeat.insertBelowScrollPosition', false);
-context.set('slots.repeatable_boxad_1.repeat.disablePushOnScroll', true);
-
-if (utils.queryString.get('recirculation_disabled') === '1') {
```
Comment (dmnsobczak) [inline examples/slots/rotating-slots/script.ts:13]: Delete only this part.

File: examples/styles.scss [hunk 3]
```diff
@@ -183,66 +177,6 @@ pre {
 	position: static;
 }
 
-.recirculation-rail {
```
Comment (dmnsobczak) [inline examples/styles.scss:186]: These are only example page CSS - also really useful 🙈

Comment (Loczi94) [reply examples/styles.scss]: Nooo, why? 😄

**Pull request merged**